lib/devise/models/rememberable.rb

```diff
@@ -100,6 +100,10 @@ def rememberable_options
       def after_remembered
       end
 
+      def remember_expired?
+        remember_created_at.nil? || (remember_expires_at <= Time.zone.now.utc)
```

Comment:
This is not exactly what I meant here: https://github.com/plataformatec/devise/pull/4926#pullrequestreview-155670776. Your change will end up returning the same value as before.

Consider the following:
```suggestion
        remember_created_at.nil? || (remember_expires_at <= Time.zone.now)
```

So the application will check reminders accordingly to local timezone.